src/NUnitEngine/nunit.engine.core/Internal/PathUtils.cs

```diff
@@ -166,6 +166,56 @@ public static string Combine(string path1, params string[] morePaths)
             return result;
         }
 
+        /// <summary>
+        /// Returns a value that indicates whether the specified file path is absolute or not on Windows operating systems.
+        /// </summary>
+        /// <param name="path">Path to check</param>
+        /// <returns><see langword="true"/> if <paramref name="path"/> is an absolute or UNC path; otherwhise, false.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="path"/></exception>
```

Comment:
```suggestion
        /// <exception cref="ArgumentNullException"><paramref name="path"/> is <see langword="null"/>.</exception>
```

Just to match the format of your other doc-comments. 🙂